### PR TITLE
Create RVRS.Load_VIP_DeathRegistrationGeneralPr.sql

### PR DESCRIPTION
**Pull request by sailendrasingh1**
01/24/2023

### CODE REVIEW

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathRegistrationGeneralPr.sql [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+	
+SET @LastLoadedDate=(SELECT MAX(LastLoadDate) FROM [RVRS_testdb].[RVRS].[Execution] WITH(NOLOCK) WHERE Entity='DeathRegistrationGeneral' AND ExecutionStatus='Completed')
+	        IF @LastLoadedDate IS NULL SET @LastLoadedDate = '01/01/1900'
+PRINT '2'  + CONVERT (VARCHAR(50),GETDATE(),109)
+	
+	
+			
+
+		        SELECT    D.DEATH_REC_ID AS SrId
+					  ,P.PersonId ,RECORD_REGIS_DATE RegistrationDate,COALESCE(RECORD_REGISTRAR,'NULL') RegistrarName,RECORD_REGIS_NUM RegistrationNumber,DATE_OF_AMENDMENT AmendmentDate,COALESCE(IND_ACCESS_STATUS,'NULL') RecordAccessId,FL_UPDATE_PENDING FlUpdatePending,AMEND_IN_PROCESS FlAmendmentInProcess,VRV_REGISTERED_FLAG FlRegistered,IND_RECORD_OWNER RecordOwner,RECORD_REGISTRAR_ID ScrRegistererUserId,FL_DELAYED FlDelayed,FL_AMENDED FlAmended,COALESCE(IND_REGIS_STATUS,'NULL') RegistrationStatusId,VRV_REC_REPLACE_NBR VRV_REC_REPLACE_NBR,OCCUR_REGIS_DATE OCCUR_REGIS_DATE,OCCUR_REGIS_NAMEL OCCUR_REGIS_NAMEL,OCCUR_REGIS_NUM OCCUR_REGIS_NUM,DOD DOD,ST_REGIS_DATE ST_REGIS_DATE
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathRegistrationGeneralPr.sql:135]: change this from  
('IND_ACCESS_STATUS',1,'','','RecordAccess**Id**', 'VARCHAR(128)',NULL,'''NULL'''),
to 
('IND_ACCESS_STATUS',1,'','','RecordAccess', 'VARCHAR(128)',NULL,'''NULL'''),

('IND_REGIS_STATUS',1,'','','RegistrationStatus**Id**', 'VARCHAR(128)',NULL,'''NULL'''),
to 
('IND_REGIS_STATUS',1,'','','RegistrationStatus', 'VARCHAR(128)',NULL,'''NULL'''),

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathRegistrationGeneralPr.sql [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+	        ,CASE WHEN RecordOwner = 'RV' AND ST_REGIS_DATE<>RegistrationDate THEN 'RecordOwner,ST_REGIS_DATE,RegistrationDate|Error:If the Record Owner is the State then Registration Date should match with State Registration Date' ELSE '' END AS LoadNote_18
+	        ,CASE WHEN ScrRegistererUserId IS NULL THEN 'ScrRegistererUserId|Error:ScrRegistererUserId should not be blank for registered records' ELSE '' END AS LoadNote_19
+	        ,CASE WHEN ISNULL(FlDelayed,'Y') NOT IN ('N','Y') THEN 'FlDelayed|Error:Not a valid value for FlUpdatePending' ELSE '' END AS LoadNote_20
+	        ,CASE WHEN ISNULL(FlAmended,1) NOT IN (0,1) THEN 'FlAmended|Error:Not a valid value for FlAmended' ELSE '' END AS LoadNote_21
+	        ,CASE WHEN (FlAmended= 1 AND VRV_REC_REPLACE_NBR = 0) OR (FlAmended = 0  AND VRV_REC_REPLACE_NBR <> 0) THEN 'FlAmended,VRV_REC_REPLACE_NBR|Error:Not a valid value for FlAmended for its version' ELSE '' END AS LoadNote_22
+					
+		INTO #Tmp_HoldData_Final
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathRegistrationGeneralPr.sql:246]: removed: 
**FlAmended = 0  AND VRV_REC_REPLACE_NBR <> 0** from LoadNote_22

**Review by almabekwoldesenbet — CHANGES_REQUESTED** [review]
As the log is big can you confirm it with Avinash

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathRegistrationGeneralPr.sql (hunk 1 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathRegistrationGeneralPr.sql:135]: fixed

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathRegistrationGeneralPr.sql (hunk 2 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathRegistrationGeneralPr.sql:246]: removed

**Review by almabekwoldesenbet — APPROVED** [review]

**Pull request merged**